src/me/goddragon/teaseai/api/session/StrokeHandler.java

```diff
@@ -42,6 +45,10 @@ public void startMetronome(int bpm, int durationSeconds) {
         currentBPM = bpm;
 
         (metronome = new Metronome()).start(bpm);
+        
+        if (TeaseAI.application.ESTIM_ENABLED.getBoolean() && TeaseAI.application.ESTIM_METRONOME.getBoolean()) {
```

Comment:
Wait there is an EstimAPI instance in the Session already. Why not use that one?

Reply:
It wasn't the same class. Renamed it to EstimMetronome to avoid confusion.